src/Google.Events.Protobuf/ProtobufJsonCloudEventFormatter.cs

```diff
@@ -28,19 +28,40 @@ namespace Google.Events.Protobuf
     /// <summary>
     /// <see cref="CloudEventFormatter"/> which is able to serialize protobuf messages in JSON.
     /// (Note that this does not implement the protobuf CloudEvent format, which is separate.)
+    /// Additionally, when deserializing binary mode events, if the data content type has a media type of "application/protobuf",
+    /// this will deserialize the body as protobuf binary data.
     /// </summary>
     /// <typeparam name="T">The type of message to serialize/deserialize</typeparam>
     public class ProtobufJsonCloudEventFormatter<T> : CloudEventFormatter where T : class, IMessage<T>, new()
```

Comment:
Maybe I'm beeing too picky, but could we deprecate this type with Json in its name, and add a new one called ProtobufCloudEventFormatter?
Or am I missing something?

Reply:
That could be a little misleading, as it might sound like it's about the Protobuf event format.

I suggest we keep the name for now, but reconsider it at the next major version. (Users don't generally use this directly anyway, so hopefully the slight oddity here won't be a problem.)

Reply:
OK sounds good!